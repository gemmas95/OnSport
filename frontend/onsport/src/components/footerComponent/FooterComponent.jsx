import React from 'react';
import './footerComponent.scss';

function FooterComponent() {
    return (
        <div className="footer__component">
            <p>By: Gemma Sanz</p>
        </div>
    );
}

export default FooterComponent;
