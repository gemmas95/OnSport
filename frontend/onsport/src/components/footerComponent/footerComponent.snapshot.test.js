import renderer from 'react-test-renderer';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import FooterComponent from './FooterComponent';

describe('FooterComponent', () => {
    const footerComponentTree = renderer.create(
        <Router>
            <FooterComponent />
        </Router>
    );
    it('should match', () => {
        expect(footerComponentTree).toMatchSnapshot();
    });
});
