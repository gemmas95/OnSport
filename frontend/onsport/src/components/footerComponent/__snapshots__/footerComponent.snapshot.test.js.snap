// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent should match 1`] = `
<div
  className="footer__component"
>
  <p>
    By: Gemma Sanz
  </p>
</div>
`;
